homeassistant/components/onewire/config_flow.py

```diff
@@ -201,7 +201,7 @@ async def async_step_configure_device(
             description_placeholders={"sensor_id": self.current_device},
         )
 
-    async def _update_options(self) -> FlowResult:
+    def _update_options(self) -> FlowResult:
```

Comment:
I think this should be categorized as a callback and better to also prefix it with `async`
```suggestion
    @callback
    def _async_update_options(self) -> FlowResult:
```